### PR TITLE
 feat(log): add justice common log format

### PR DESCRIPTION
**Pull request by AdityaDerrySaputra**
feat(log): add justice common log format

### CODE REVIEW

File: justice_python_common_log/flask.py [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+        data = {
+            "time" : datetime.now(timezone.utc).strftime('%Y-%m-%dT%H:%M:%SZ'),
+            "realm" : os.getenv("REALM", "dev"),
+            "method" : request.method,
+            "path" : request.path,
+            "status" : response.status_code,
+            "duration" : int((datetime.now() - g.start).total_seconds())
+        }
+
+        if bool(strtobool(os.getenv("FULL_ACCESS_LOG_ENABLED", FULL_ACCESS_LOG_ENABLED))):
```
Comment (mhanifsp) [inline justice_python_common_log/flask.py]: any reason to bool after strtobool?

**Review by mhanifsp — APPROVED** [review]

File: justice_python_common_log/flask.py (hunk 1 above)
Comment (AdityaDerrySaputra) [reply justice_python_common_log/flask.py]: ohh i miss it, thank you. updated

**Review by ghiffari-agsarya — APPROVED** [review]

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+with open('README.rst') as readme_file:
+    readme = readme_file.read()
+
+with open('HISTORY.rst') as history_file:
+    history = history_file.read()
+
+requirements = ['pyjwt[crypto]==2.2.0']
+
+optional_requirements = {
```
Comment (NurFaizin) [inline setup.py:17]: This variable is not used anywhere?

File: setup.py [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    description="Justice common log format for python",
+    install_requires=requirements,
+    license="Apache Software License 2.0",
+    long_description=readme + '\n\n' + history,
+    include_package_data=True,
+    keywords='justice_python_common_log',
+    name='justice_python_common_log',
+    packages=find_packages(include=['justice_python_common_log', 'justice_python_common_log.*']),
+    test_suite='tests',
+    tests_require=test_requirements,
```
Comment (NurFaizin) [inline setup.py:45]: test_requirements is not defined?

File: setup.py (hunk 2 above)
Comment (AdityaDerrySaputra) [reply setup.py:17]: Done

File: setup.py (hunk 3 above)
Comment (AdityaDerrySaputra) [reply setup.py:45]: Done

**Review by NurFaizin — APPROVED** [review]

**Pull request merged**